presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -303,6 +303,39 @@ KHyperLogLog
     A KHyperLogLog is a data sketch that can be used to compactly represents the association of two
     columns. See :doc:`/functions/khyperloglog`.
 
+SetDigest
+---------
+
+.. _setdigest_type:
+
+``SetDigest``
+^^^^^^^^^^^^^
+
+A SetDigest (setdigest) is a data sketch structure used
+in calculating `Jaccard similarity coefficient <https://wikipedia.org/wiki/Jaccard_index>`_
+between two sets.
+
+SetDigest encapsulates the following components:
+
+- `HyperLogLog <https://wikipedia.org/wiki/HyperLogLog>`_
+- `MinHash with a single hash function <http://wikipedia.org/wiki/MinHash#Variant_with_a_single_hash_function>`_
+
+HyperLogLog: This is a probabilistic data structure used for the approximation of the number of distinct elements in
+a large dataset. It is based on the principle of observing the 'rarity' of certain patterns in a random process.
+HyperLogLog provides a way to estimate the number of distinct elements (the cardinality) with a high degree
+of accuracy, using a relatively small amount of memory.
+
+MinHash with a single hash function: This is a technique used to quickly estimate the similarity between two sets.
+It involves creating a 'signature' or 'sketch' for each set, which is a much smaller subset that still maintains the
```

Comment:
The words 'signature' and 'sketch' are enclosed in single vertical quotes. This is not RST formatting. Is this content copied from somewhere else? If so, it should be attributed.

Why are the words marked like this?
What do you mean by this?
Is this marking needed, or can it be removed or changed?